airflow/migrations/env.py

```diff
@@ -21,7 +21,7 @@
 from alembic import context
 
 from airflow import models, settings
-from airflow.models.serialized_dag import SerializedDagModel  # noqa
+from airflow.models.serialized_dag import SerializedDagModel  # pylint: disable=unused-import # noqa
```

Comment:
@potiuk Do you remember if we need this import there?

I remember you were moving the imports for the migrations to work, so I am not sure

Reply:
It was not here (it was where reset method was) but I think for migrations it's also needed - in order to generate migration diffs for all the model objects.

Reply:
Possibly we even need to add more here (from airflow.utils.db). Possibly one place where we import one modes?:

```
from airflow.models import (  # noqa: F401 # pylint: disable=unused-import
    DAG, XCOM_RETURN_KEY, BaseOperator, BaseOperatorLink, Connection, DagBag, DagModel, DagPickle, DagRun,
    DagTag, Log, Pool, SkipMixin, SlaMiss, TaskFail, TaskInstance, TaskReschedule, Variable, XCom,
)
# We need to add this model manually to get reset working well
# noinspection PyUnresolvedReferences
from airflow.models.serialized_dag import SerializedDagModel  # noqa: F401  # pylint: disable=unused-import
```

Reply:
I think it might be ok, we only needed that in `db.resetdb`. Ran it locally, didn't error

https://github.com/apache/airflow/pull/7381
https://github.com/apache/airflow/pull/6938/files

Reply:
WDYT @potiuk ?

Reply:
LGTM!

Reply:
In that case, can you approve it :D